e2e/cypress/integration/login/tutorial_navigation_and_links_spec.js

```diff
@@ -0,0 +1,149 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+// ***************************************************************
+// - [#] indicates a test step (e.g. 1. Go to a page)
+// - [*] indicates an assertion (e.g. * Check the title)
+// - Use element ID when selecting an element. Create one if none.
+// ***************************************************************
+
+/* eslint max-nested-callbacks: ["error", 4] */
+
+describe('Test Tutorial Navigation', () => {
+    before(() => {
+        cy.loginAsNewUserWithTutorial();
+    }); // before
+
+    it('MM-13989 / On13989: After login, the first page of the tutorial should display.', () => {
```

Comment:
We should only need the `On13989` ID included in the name.

Reply:
Removed MM-13989 and /